engine.json.minimum

```diff
@@ -17,6 +17,9 @@
     "spark.kryoserializer.buffer.mb": "300",
     "spark.kryoserializer.buffer": "300m",
     "spark.executor.memory": "4g",
+    "spark.executor.cores": "2",
+    "spark.task.cpus": "2",
+    "spark.default.parallelism": "8",
     "es.index.auto.create": "true"
   },
   "algorithms": [
```

Comment:
I didn't notice any appreciable gains beyond a certain level, which was ~3x-4x the number of executors. With too many partitions, you'll spend more time in the first and last stages on I/O, and not see any real gain on the individual tasks. I ended up going with 4 executors, each with 2 cores, and a default parallelism of 16.

I agree a tuning example would be better. I'll get the change made shortly.